homeassistant/components/shelly/climate.py

```diff
@@ -238,19 +237,17 @@ async def set_state_full_path(self, **kwargs: Any) -> Any:
         """Set block state (HTTP request)."""
         LOGGER.debug("Setting state for entity %s, state: %s", self.name, kwargs)
         try:
-            async with async_timeout.timeout(AIOSHELLY_DEVICE_TIMEOUT_SEC):
-                return await self.coordinator.device.http_request(
-                    "get", f"thermostat/{self._channel}", kwargs
-                )
-        except (asyncio.TimeoutError, OSError) as err:
-            LOGGER.error(
-                "Setting state for entity %s failed, state: %s, error: %s",
-                self.name,
-                kwargs,
-                repr(err),
+            return await self.coordinator.device.http_request(
+                "get", f"thermostat/{self._channel}", kwargs
             )
-            self.coordinator.last_update_success = False
```

Comment:
How is the coordinator last update success handled with the change?

Reply:
Not handled, it should set be set to false, fixed